server/src/main/kotlin/fr/androidmakers/server/CachedData.kt

```diff
@@ -116,3 +135,11 @@ object CachedData {
         return emptyList()
     }
 }
+
+fun String.encodeBase64(): String {
+    return Base64.getEncoder().encodeToString(toByteArray())
+}
+
+fun String.decodeBase64(): String {
+    return String(Base64.getDecoder().decode(this))
+}
```

Comment:
Why encode/decode to base64?

Reply:
It's not really needed but I think it's a good practice for them to be "opaque" - it conveys the fact that they're really cursors and not ids 😊.  Some [reference](https://graphql.org/learn/pagination/#:~:text=In%20general%2C%20we%27ve,base64%20encoding%20them.).